NDTensors/ext/examples/Readme.md

```diff
@@ -2,6 +2,7 @@ In this file I will layout how to construct a new backend NDTensor datatype stor
 
 One should start by constructing a folder designated `NDTensor___` where ___ is replaced with the name of the datatype. In this folder one creates an `NDTensor___.jl` file. This is the only file which should link `NDTensors` to your new datatype.  In this file, one can import all necessary packages for your datatype. These packages should NOT be added to the `Project.toml`.
 
+## TODO This is no longer necessary, should it be deleted?
```

Comment:
As I point out in this comment, I don't think we will need to have users define the `set_eltype` and `set_ndims` functions for different backends (though I guess they will need to define the `SetParameters` functions from `interface.jl`, correct? In principle, since we have the `unset_parameters` function now across our known types, we could start implementing a compiled system to make `set_parameter` functions for different values of `nparameters` too right? After a generic system is implemented that would mean users just need to define a few functions like `nparameters`, `default_parameter` for each index value, and `unset_parameters`.

Reply:
Good point, I was thinking the same thing. It does seem like we can make a generic version of `set_parameters` based on `nparameters` and `unset_parameters`.

Reply:
To summerize, I will make a minimal system using `parenttype_position` for the small number of types and will create a generic `set_ndim` and `set_eltype` function that uses this `parenttype_position` function. I will also revert the push `601dc652244bf06be59dd7bc0ee8cf5649640a85`